ocaml/xapi-idl/xen/xenops_interface.ml

```diff
@@ -219,11 +219,7 @@ end
 module Vgpu = struct
   include Xenops_types.Vgpu
 
-  type implementation =
-    | GVT_g of gvt_g
```

Comment:
Can't this be a problem when upgrading?

Reply:
> So long, old friend 👋

So long, old friend 😄

Reply:
It shouldn't be, 
The VM will be shutdown during RPU, 
After upgrade, xapi will block VM start if the VM has this vGPU assigned, then it will not go this far.